### ISSUE TITLE
New Flake8 exception?

### ISSUE THREAD

**Issue by itsmehemant123** [LINKIWI/modern-paste#58]
Hello!

I've attempted to install on my ubuntu box. I've had two issues doing so:
- My global python was 2.7, but the `pip` was still pointing to 3.5, so I had to switch the command in the makefile from `pip install -r requirements.txt` to `python -m pip install -r requirements.txt`

On fixing this, I had another issue during the Flake8 run -

```
app/api/user.py:57:5: E722 do not use bare except'
app/api/user.py:98:5: E722 do not use bare except'
app/api/user.py:115:5: E722 do not use bare except'
app/api/user.py:131:5: E722 do not use bare except'
app/api/user.py:147:5: E722 do not use bare except'
app/api/user.py:163:5: E722 do not use bare except'
app/util/cryptography.py:40:5: E722 do not use bare except'
app/util/cryptography.py:68:5: E722 do not use bare except'
app/util/cryptography.py:76:5: E722 do not use bare except'
app/views/paste.py:105:5: E722 do not use bare except'
app/api/paste.py:90:5: E722 do not use bare except'
app/api/paste.py:123:5: E722 do not use bare except'
app/api/paste.py:152:5: E722 do not use bare except'
app/api/paste.py:190:5: E722 do not use bare except'
app/api/paste.py:209:5: E722 do not use bare except'
app/api/paste.py:228:5: E722 do not use bare except'
app/api/paste.py:247:5: E722 do not use bare except'
```

I've added E722 as an exception to the `setup.cfg`, but I'm not sure why this occurred. Is it only me, or anyone else facing the same issue?

**Comment by tgunr** [comment]
Same issue here

**Comment by LINKIWI** [comment]
They are just lint/style warnings; it will not affect program behavior and you can safely ignore them.

**Comment by itsmehemant123** [comment]
Yes, I've added those to the exceptions and got it working. I was wondering if they were supposed to be working before, and recently started breaking the build process.

Thanks for the great work though!

**Comment by LINKIWI** [comment]
I didn't pin a version of flake8 in the dependencies declaration so it's most likely the case that there was a breaking change released some time in the last couple of years that caused new errors to arise. I haven't put any effort toward fixing them since I'm focusing work on Modern Paste v2  (not yet open source).

**Comment by tgunr** [comment]
Here is a quick summary of what it took me to get running on a fresh Debian VM

http://confluence.polymicro.net/confluence/display/POLY/2018/04/06/Modern+Paste
